<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFormEquipo</class>
 <widget class="QDialog" name="DialogFormEquipo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario de Equipo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
   QDialog {
       background-color: rgb(113, 177, 129);
   }
   QLabel#lblTitulo {
       font-size: 22pt;
       font-weight: bold;
       color: white;
       background-color: rgb(1, 107, 97);
       border-radius: 8px;
       padding: 10px;
   }
   QPushButton {
       background-color: rgb(1, 107, 97);
       color: white;
       border-radius: 8px;
       padding: 8px;
       font-weight: bold;
   }
   QPushButton:hover {
       background-color: #93da97;
       color: rgb(1, 107, 97);
   }
   QPushButton:pressed {
       background-color: rgb(1, 107, 97);
   }
   QLineEdit, QComboBox {
       border: 2px solid rgb(1,107,97);
       border-radius: 5px;
       padding: 4px;
       background-color: white;
       color: black;
   }
   QLabel#lblVistaPrevia {
       border: 2px solid rgb(1,107,97);
       border-radius: 8px;
       background-color: white;
   }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitulo">
     <property name="text">
      <string>DATOS DEL EQUIPO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Informaci√≥n del Equipo</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblNombre">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nombre del Equipo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtNombre">
        <property name="placeholderText">
         <string>Ej: Los Tigres</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblCurso">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Curso:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbCurso"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblColor">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Color de Camiseta:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="txtColor">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Seleccionar color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSeleccionarColor">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Seleccionar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogFormEquipo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogFormEquipo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
